docs/GraFx-Developers/connectors/authorization-for-connectors/index.md

```diff
@@ -0,0 +1,287 @@
+
+!!! note "Authorization Always Happens on the Server"
+
+	  You can define different authorization types for each usage scenario. However, the actual authorization always occurs on the server to prevent token or credential leakage.
+
+!!! note "Each Usage Can Only Be Set Once"
+
+	  If you run the `set-auth` command twice with the same authorization usage `-au` value but different `-at` authorization types, the authorization type will just be ignored. At this moment there is no way to overwrite or erase authorization once set.
+
+### Authorization Type (`-at`)
+
+This arguments expects one of the four supported types of authorization.
+
+| Auth Type | Reference | Usage Support | Expected Argument Value |
+|-----------|-----------|---------------|---------------------|
+| Static Header Key | [MDN - Authorization Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) | browser and server | staticKey |
+| OAuth Client Credentials | [OAuth - Client Credentials](https://oauth.net/2/grant-types/client-credentials/) | browser and server | oAuth2ClientCredentials |
+| OAuth Authorization Code | [OAuth - Authorization Code](https://oauth.net/2/grant-types/authorization-code/) | browser | oAuth2AuthorizationCode |
+| OAuth Resource Owner Password | [OAuth - Password Grant](https://oauth.net/2/grant-types/password/) | browser and server | oAuth2ResourceOwnerPassword |
+
+In addition to specifying the authorization type using the `-at` argument, you must also define the supported authorization types in your `package.json` file. See [Define Supported Authorization Types](#define-supported-authorization-types)
+
+!!! warning "Each Usage Can Only Be Set Once"
+
+    OAuth Authorization Code is only supported with browser usage.
+
+<br/>
+
+#### OAuth Considerations
+
+1. OAuth responses must comply with the [Access Token Response](https://www.oauth.com/oauth2-servers/access-tokens/access-token-response/) format, with the exception `expires_in` being mandatory.
+2. The `grant_type` for OAuth types is automatically set as per the specification and cannot be configured.
+3. `access_token` and `refresh_token` are cached internally for OAuth types. There's currently no way to reset them without deleting the Connector.
+4. OAuth Authorization Code is user-specific and not useful with an Integration User (used outside the GraFx Platform).
+5. OAuth Authorization Code is limited to "browser" usage, which may restrict its applicability in certain scenarios.
+
+### Authorization Data File Requirements (`--auth-data-file`)
+
+Each authorization type requires a specific JSON schema which is related to the requirements of that authorization type. Keep in mind that the `grant_type` for OAuth types is predefined by the specifications.
+
+Each authorization type requires a specific JSON schema. The `grant_type` for OAuth types is predefined by the specifications.
```

Comment:
This is a copy of the previous paragraph, just a teensy bit less detailed. Choose one to stay and one to shuffle off this mortal coil